doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -97,8 +97,7 @@ PyArray_Type and PyArrayObject
           /* version dependent private members */
       } PyArrayObject;
 
-   .. c:macro:: PyObject_HEAD
-
+   :c:macro:`PyObject_HEAD`
```

Comment:
I don't think this is right, the correct place to point a reference to PyObject_HEAD is to the [PyObject_HEAD](https://docs.python.org/3/c-api/structures.html#c.PyObject_HEAD) in the cpython documentation. Is there a warning about a missing reference somewhere?

Reply:
This reference points to cpython documentation(https://output.circle-artifacts.com/output/job/fc72e3e8-643a-471e-b9d6-c2bf1eda06f4/artifacts/0/doc/build/html/reference/c-api/types-and-structures.html#c.NPY_AO), with no warning about it.